.genx/templates/react/grid.hbs

```diff
@@ -1,5 +1,7 @@
 {{#if config.permissions.viewRight}}hasUserPermission('{{config.permissions.viewRight}}') ? ({{/if}}
 <rapid-grid-pro
+  header-case-type="capitalCase"{{#if config.useOnlyTemplateCols}}
```

Comment:
nit: any chance we can move the #if to the next line (or any issues?)

Reply:
This would result in an extra blank line in the output file